### ISSUE TITLE
language selection on login screen doesn't work when user has not yet signed in

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#2594]
### Observed behavior

Trying to switch languages on the login screen fails. If it's in English and you switch to French, it stays in English

### Expected behavior

Language should switch

### User-facing consequences

Non-english speakers are sad

### Errors and logs

```
[13/Nov/2017 14:50:30] "POST /i18n/setlang/ HTTP/1.1" 403 2629
```

### Context

Kolibri 0.7

**Comment by christianmemije** [comment]
Hmm could not reproduce
![lang](https://user-images.githubusercontent.com/7193975/32758415-222c53c2-c899-11e7-952f-3a5291fa2f9f.gif)

**Comment by christianmemije** [comment]
Update. Could reproduce by opening in incognito. Might be due to the missing `django_language` cookie @rtibbles?

**Comment by christianmemije** [comment]
@indirectlylit Reproduced on 0.6.x.

**Comment by indirectlylit** [comment]
if this is only incognito, I'll reduce the priority. thanks for investigating!

**Comment by christianmemije** [comment]
Not *only* in incognito, that is just a way to reproduce. Will also occur on a device where a sign in has not occurred.

**Comment by mrpau-richard** [comment]
@indirectlylit & @christianmemije can I take over this issue?
I reproduced this issue when I open it in the different browsers, After I created an admin account.  
I used the latest [pre-release](https://github.com/learningequality/kolibri/releases) of Kolibri to test this issue.

**Comment by indirectlylit** [comment]
absolutely! feel free to grab any 0.7 issues without an assignee. thanks!

**Comment by rtibbles** [comment]
Note that this also affects language selection in the setup wizard, and language selection when accessing Kolibri as guest.

**Comment by rtibbles** [comment]
Fixed in #2748
